src/Integration/Binding/INuGetBindingOperation.cs

```diff
@@ -43,6 +43,6 @@ internal interface INuGetBindingOperation
         /// Extracts any required information from the supplied Roslyn export profile
         /// </summary>
         /// <returns>True if processing was successful, otherwise false</returns>
-        bool ProcessExport(Language language, RoslynExportProfileResponse exportProfileResponse);
+        bool ProcessExport(Language language, IEnumerable<NuGetPackageInfo> nugetPackages);
```

Comment:
This is the main part of the change: we aren't going to use the `RoslynExportProfileResponse` any more, so the the NuGet binding interfaces need to use a different type.